src/main/java/com/epam/eighty/repository/CustomerRepository.java

```diff
@@ -13,11 +14,11 @@
 @Repository("customerRepository")
 public interface CustomerRepository extends BaseRepository<Customer, Long> {
 
-    @Query(value = "MATCH (customer:`Customer`) WHERE customer.name =~{0} RETURN customer", elementClass = Customer.class)
-    Slice<Customer> getSortedSetOfCustomersByName(String customerName);
```

Comment:
I would not to use the word "Sorted" in method's name as well as the word "set". GetCustomersByName is quite good method's name IMHO.